File: src/cs/fizzbuzz.cs
using System;

class Program
{
	public static void Main()
	{
        for (var i = 1; i <= 100; i++){
            if (i % 3 == 0 && i % 5 == 0)
                Console.Write("FizzBuzz");
            else if (i % 3 == 0)
                Console.Write("Fizz");
            else if (i % 5 == 0)
                Console.Write("Buzz");
            else
                Console.Write(i);

            if (i % 10 == 0)
                Console.WriteLine();
            else
                Console.Write(",");
        }
	}
}

File: src/vb/fizzbuzz.vb
Imports System

Public Module Program
    Sub Main()

        For i As Integer = 1 To 100
            If i Mod 3 = 0 And i Mod 5 = 0
                Console.Write("FizzBuzz")
            ElseIf i Mod 3 = 0
                Console.Write("Fizz")
            ElseIf i Mod 5 = 0
                Console.Write("Buzz")
            Else
                Console.Write(i)
            End If

            If i Mod 10 = 0
                Console.WriteLine
            Else
                Console.Write(",")
            End If
        Next
   End Sub
End Module